.github/workflows/dynamiclinks.yml

```diff
@@ -3,8 +3,8 @@ name: dynamiclinks
 on:
   pull_request:
```

Comment:
Undo changes in this file - `git checkout master -- .github/workflows/dynamiclinks.yml`